README.md

```diff
@@ -1,25 +1,33 @@
 # Express-log-to-json [![Build Status](https://secure.travis-ci.org/noblesamurai/express-log-to-json.png?branch=master)](http://travis-ci.org/noblesamurai/express-log-to-json) [![NPM version](https://badge-me.herokuapp.com/api/npm/express-log-to-json.png)](http://badges.enytc.com/for/npm/express-log-to-json)
```

Comment:
Not sure this is a very good name. Not very descriptive of what the module does.

Perhaps `express-request-to-json`?